packages/core/src/App/Containers/AccountSwitcher/account-switcher-account-wrapper.jsx

```diff
@@ -1,12 +1,14 @@
 import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Icon } from '@deriv/components';
+import { Icon, Text } from '@deriv/components';
 
 const AccountWrapper = ({ children, header, is_visible, toggleVisibility }) => (
     <React.Fragment>
         <div className={classNames('acc-switcher', { 'acc-info--show': !is_visible })} onClick={toggleVisibility}>
-            <span className='acc-switcher__list-title'>{header}</span>
+            <Text size='s' weight='bold' align='left' color='prominent' lineHeight='l' flex>
```

Comment:
What is the purpose of `flex` here?

Reply:
It was on the previous style and I kept it, but we decided not to add it to the text component, so I will remove it and put it on the container.